.buildkite/filebeat/filebeat-pipeline.yml

```
@@ -1,5 +1,139 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
+  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  IMAGE_WIN_2016: "family/core-windows-2016"
+  IMAGE_WIN_2019: "family/core-windows-2019"
+  IMAGE_WIN_2022: "family/core-windows-2022"
+  IMAGE_MACOS_13: "generic-13-ventura-x64"
```

If possible use env variable name without the version:

```suggestion
  IMAGE_MACOS: "generic-13-ventura-x64"
```

---

@oakrizan let's use the equivalent done in metricbeat: https://github.com/elastic/beats/pull/37592/files#diff-87cc9e7aecd3c1a6382e951a1b17429443aa3f12387328521fa06b2c8f83a814R8